frontend/src/images/experiment-type-addon.svg

```diff
@@ -0,0 +1,4 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 16 16"><path d="M14.53,8c-1,0-1,1-1.75,1A.77.77,0,0,1,12,8.25V5a1,1,0,0,0-1-1H7.78A.77.77,0,0,1,7,3.25C7,2.5,8,2.47,8,1.5S7.13,0,6,0,4,.64,4,1.5s1,1,1,1.75A.77.77,0,0,1,4.28,4H1A1,1,0,0,0,0,5V7.25A.77.77,0,0,0,.78,8c.75,0,.78-1,1.75-1S4,7.9,4,9s-.64,2-1.5,2-1-1-1.75-1a.77.77,0,0,0-.75.75V15a1,1,0,0,0,1,1H4.28A.77.77,0,0,0,5,15.25c0-.75-1-.78-1-1.75S4.92,12,6,12s2,.64,2,1.5-1,1-1,1.75a.77.77,0,0,0,.75.75H11a1,1,0,0,0,1-1V11.75a.77.77,0,0,1,.75-.75c.75,0,.78,1,1.75,1s1.5-.9,1.5-2S15.39,8,14.53,8Z" fill="context-fill"/></svg>
```

Comment:
I just copied these SVG images over from #2546 - I don't know SVG well enough to know what you're asking for here as a change. Are there different images I should use?